sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1476,8 +1477,13 @@ public void close() throws IOException {
      * Writes just the values to Kafka. This is useful for writing collections of values rather
      * thank {@link KV}s.
      */
+    @SuppressWarnings({ "unchecked" })
     public PTransform<PCollection<V>, PDone> values() {
-      return new KafkaValueWrite<>(toBuilder().build());
+      return new KafkaValueWrite<>(
+              toBuilder()
```

Comment:
The indentation seems off. Can you make 2? (or 4).

Reply:
Hows this? I tried to match indentations I could see in the file with a 4space from the start of prev line. But I'm not seeing a constancy, with function params, within the whole of the file to make a sound judgment it's right.